thrift/src/main/java/com/linecorp/armeria/server/thrift/ThriftDocServicePlugin.java

```diff
@@ -168,7 +224,8 @@ static ServiceInfo newServiceInfo(
             });
         }
 
-        return new ServiceInfo(name, functions, classes, endpoints, docStrings.get(name), exampleHeaders);
+        return new ServiceInfo(name, functions, classes, endpoints,
```

Comment:
After thinking a little bit more, it seems to me that there's no need to make this 'docStringId' public.